plugins/sublime/sublime.plugin.zsh

```diff
@@ -1,4 +1,4 @@
-if [[ $('uname') == 'Linux' ]]; then
+if [[ "$OSTYPE" == linux-gnu* ]]; then
```

Comment:
I changed this to `*-gnu*`, because this plugin is really looking for a GNU env, not a Linux env. Setting this incorrectly would break kFreeBSD, which has a BSD kernel and GNU userland.